cmd/yurthub/app/config/config.go

```diff
@@ -308,6 +308,12 @@ func registerInformers(options *options.YurtHubOptions,
 		informerFactory.InformerFor(&corev1.Secret{}, newSecretInformer)
 	}
 
+	if workingMode == util.WorkingModeCloud {
+		newPodInformer := func(client kubernetes.Interface, resyncPeriod time.Duration) cache.SharedIndexInformer {
+			return coreinformers.NewPodInformer(client, "", resyncPeriod, nil)
```

Comment:
how about set the list options to a fieldSelector `obj.Spec.NodeName=nodeName`? so only pods for this node will be returned and efficiency can be improved.